app/forms/umrdr/forms/work_form.rb

```diff
@@ -31,5 +29,14 @@ def rights
         @model.rights.first
     end
 
+    # You like danger?! Because you better pass in a hash with the correct keys. 
+    # Can be alleviated when form coverage attribute is:
+    # `coverages: [{begin: {:year, :month, :day}, end: {:year, :month, :day}}, ... ]`
+    def merge_date_coverage_attributes!(hsh)
+      hsh ||= {}
+      hsh.stringify_keys!
+      @attributes.merge! hsh
+    end
```

Comment:
Good catch on the not mutating the hsh.  The hsh arg will be nil or a hash.  The lonely operator is tempting.  Probably going to go with that one.